src/adera_app/feature_interfaces.h

```diff
@@ -76,56 +76,63 @@ inline osp::PipelineTypeInfo const gc_infoForEStgIntr
 /**
  * @brief Continuous Containers, data that persists and is modified over time
  *
+ * This pipeline is intended to organize access to variables associated with entities that can be
+ * added and removed.
+ *
+ * Things that were considered (about game engines and simulations in general):
+ * * When a new entity is created, it should NOT be moved (physics updated) right away before it is
+ *   first rendered. It should not appear offset from where it was requested to be initialized.
+ * * It shouldn't be possible to create and immediately delete an entity on the same frame before
+ *   it is rendered. The entity should live at least one frame for other systems to be able to
+ *   process and respond to entities being added or removed.
+ * * Creating an entity and simultaneously requesting a floating origin translation is tough case.
+ *   Make sure all positions are translated before requesting to spawn new entites.
+ *
+ * More explainations on the current order:
+ * * Modify comes before Delete and New, because it's janky to modify elements directly after
+ *   they're added. It only cares about 'what currently exists'.
+ * * ReadyB4New is there to read results of Modify, without considering new or deleted elements.
+ *   Elements can be requested to be deleted here. This also handles the case for requesting a
+ *   floating origin translation like mentioned above.
+ * * Delete runs before New, because Delete makes empty spaces in buffers for new elements to fill
+ * * Resize comes before New, to resize/reallocate containers to a known new maximum size before
+ *   adding new elements.
  */
 enum class EStgCont : uint8_t
 {
-    Delete      = 0,
-    ///< Remove elements from a container or mark them for deletion. This often involves reading
-    ///< a set of elements to delete. This is run first since it leaves empty spaces for new
-    ///< elements to fill directly after
-
-    Resize_     = 1,
-    ///< Resize the container to fit more elements
-
-    New         = 2,
-    ///< Add new elements
-
-    Modify      = 3,
-    ///< Modify existing elements
-
-    ScheduleC   = 4,
-
-    Ready       = 5,
-    ///< Container is ready to use
-
-    ReadyWorkaround = 6,
+    Modify      = 0,    ///< Modify existing elements
+    ReadyB4New  = 1,    ///< Container is ready to use after modify, but before adding new elements
+    Delete      = 2,    ///< Remove elements from a container
+    Resize_     = 3,    ///< Resize the container to fit more elements
```

Comment:
Why the underscore?

Reply:
conflict with another enum named `Resize`. These are just `using enum`-ed in for readability. I thought about making individual 'stages' global variables instead, where each pipeline can list off which stages they use, so they can share the same stage names. The metaprogramming would be harder though.